enterprise-suite/gotests/tests/alertmanager/alertmanager_test.go

```diff
@@ -146,52 +174,75 @@ var _ = Describe("all:alertmanager", func() {
 
 		deleteAlert := func(name string) {
 			err := console.DeleteMonitor("es-alert-test/" + name)
-			Expect(err).ToNot(HaveOccurred())
+			Expect(err).ToNot(HaveOccurred(), "should have deleted monitor")
 		}
 
 		It("is firing in alertmanager", func() {
-			name := "alert_monitor_alertm"
+			name := "alert_monitor_alert"
 			setupAlert(name)
 
 			// Look for alert from our monitor using alertmanager api
-			alerts, err := alertm.Alerts()
-			Expect(err).ToNot(HaveOccurred())
-			var alertNames []string
-			for _, alert := range alerts {
-				if val, ok := alert.Labels["alertname"]; ok {
-					alertNames = append(alertNames, val)
+			err := util.WaitUntilSuccess(util.LongWait, func() error {
+				alerts, err := alertm.Alerts()
+				Expect(err).ToNot(HaveOccurred())
+				var alertNames []string
+				var found bool
+				for _, alert := range alerts {
+					if val, ok := alert.Labels["alertname"]; ok {
+						alertNames = append(alertNames, val)
```

Comment:
Looks like there's no more need to add firing alerts to the list if the test now uses `found` variable.

Reply:
Isn't it worth it for the logging when the test fails?

Reply:
Oh, didn't notice that at first